startup.sh

```diff
@@ -12,7 +13,8 @@ EOF
 tput sgr0
 
 # Generate PKI and config if necessary
-if test ! -d data; then
+data_version="$(./generate/get-version.sh)"
```

Comment:
If you're relying on relative paths being meaningful, you probably want to do this at the start of your script

`cd "$(dirname "$0")"`